ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -231,102 +232,121 @@ Item {
+                }
 
+                onSendMessage: {
+                    if (!d.activeChatContentModule) {
+                        console.debug("error on sending message - chat content module is not set")
+                        return
+                    }
 
-            onSendMessage: {
-                if (!d.activeChatContentModule) {
-                    console.debug("error on sending message - chat content module is not set")
-                    return
+                    if (root.rootStore.sendMessage(d.activeChatContentModule.getMyChatId(),
+                                                   event,
+                                                   chatInput.getTextWithPublicKeys(),
+                                                   chatInput.isReply? chatInput.replyMessageId : "",
+                                                   chatInput.fileUrlsAndSources
+                                                   ))
+                    {
+                        Global.playSendMessageSound()
+
+                        chatInput.textInput.clear();
+                        chatInput.textInput.textFormat = TextEdit.PlainText;
+                        chatInput.textInput.textFormat = TextEdit.RichText;
+                    }
                 }
 
-                if (root.rootStore.sendMessage(d.activeChatContentModule.getMyChatId(),
-                                              event,
-                                              chatInput.getTextWithPublicKeys(),
-                                              chatInput.isReply? chatInput.replyMessageId : "",
-                                              chatInput.fileUrlsAndSources
-                                              ))
-                {
-                    Global.playSendMessageSound()
-
-                    chatInput.textInput.clear();
-                    chatInput.textInput.textFormat = TextEdit.PlainText;
-                    chatInput.textInput.textFormat = TextEdit.RichText;
+                onUnblockChat: {
+                    d.activeChatContentModule.unblockChat()
                 }
-            }
 
-            onUnblockChat: {
-                d.activeChatContentModule.unblockChat()
+                onKeyUpPress: {
+                    d.activeMessagesStore.setEditModeOnLastMessage(root.rootStore.userProfileInst.pubKey)
+                }
             }
 
-            onKeyUpPress: {
-                d.activeMessagesStore.setEditModeOnLastMessage(root.rootStore.userProfileInst.pubKey)
+            ChatPermissionQualificationPanel {
+                id: channelPostRestrictions
+                width: (chatInput.textInput.width-Style.current.bigPadding) // by design
+                height: chatInput.textInput.height
+                anchors.left: parent.left
+                anchors.leftMargin: (2*Style.current.bigPadding)
+                visible: (!root.rootStore.permissionsStore.allTokenRequirementsMet && !!viewOnlyHoldingsModel)
```

Comment:
Same comment than before, I think that the model should be `viewAndPostHoldingsModel` instead!

Reply:
updated, please have a second look!